src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -145,11 +145,22 @@ public function getEnv($prefix, $name, \Closure $getEnv)
             $this->loaders = new \ArrayIterator();
 
             try {
-                while ((false === $env || null === $env) && $loaders->valid()) {
-                    $loader = $loaders->current();
-                    $loaders->next();
+                $index = 0;
```

Comment:
the block should be wrapped in `if (false === $env || null === $env) { }`

Reply:
good  catch